handbook/product/user_research/user_research_participant.md

```diff
@@ -0,0 +1,18 @@
+# So you’re about to participate in user testing
+
+Thank you so much for helping with our user research! Remote moderated hallway testing is a great way for us to quickly test and iterate on our ideas so that we can identify opportunities and challenges early.
```

Comment:
```suggestion
Thank you so much for agreeing to test the design! Remote moderated hallway testing is a great way for us to quickly test and iterate on our ideas so that we can identify opportunities and challenges early.
```

Helping is very general - maybe it makes sense to differentiate this one from just observing (also helping)?

Reply:
To be honest, I'd rather keep it general, because not all user research will be explicitly "tests of the design." It might be that it's just a conversation, or maybe it's another research technique that we can do over Lookback but doesn't involve testing.